src/metadata/ffmpeg_handler.cc

```diff
@@ -241,7 +241,7 @@ void FfmpegHandler::addFfmpegResourceFields(const std::shared_ptr<CdsItem>& item
     }
 
 #if defined(HAVE_FFMPEG) && defined(HAVE_FFMPEGTHUMBNAILER)
-    if (config->getBoolOption(CFG_SERVER_EXTOPTS_FFMPEGTHUMBNAILER_ENABLED) && (startswith(item->getMimeType(), "video") || item->getFlag(OBJECT_FLAG_OGG_THEORA))) {
+    if (config->getBoolOption(CFG_SERVER_EXTOPTS_FFMPEGTHUMBNAILER_ENABLED) && (item->getClass() == UPNP_CLASS_VIDEO_ITEM || item->getFlag(OBJECT_FLAG_OGG_THEORA))) {
```

Comment:
You can drop the check for the flag because it is used to detect a video

Reply:
Done.